bin/kano-feedback-cli

```diff
@@ -2,35 +2,59 @@
 
 # kano-feedback-cli
 #
-# Copyright (C) 2014, 2015 Kano Computing Ltd.
-# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU General Public License v2
-#
-# This tool allows for gathering kano-feedback data into a local tar compressed file
-#
-
+# Copyright (C) 2014-2018 Kano Computing Ltd.
+# License: http://www.gnu.org/licenses/gpl-2.0.txt GNU GPL v2
+
+"""
+kano-feedback-cli allows for gathering kano-feedback data into a
+    local tar compressed file. Requires sudo.
+
+Usage:
+    kano-feedback-cli [options]
+    kano-feedback-cli -h | --help
+
+Options:
+    -o, --output=<path>
```

Comment:
So you're changing the interface? Then the uses need to be fixed, eg: https://github.com/KanoComputing/kano-settings/blob/280a2183d5a7e36ec8d7af9615d9e80b6a4b526f/bin/kano-capture-logs#L28

Reply:
Yup, there is a PR for that also :)

Reply:
👍